build/Dockerfile.test

```diff
@@ -1,7 +1,6 @@
 FROM python:3
 
 # Install modules
-COPY src/requirements.txt ./
 ENV PYTHONDONTWRITEBYTECODE="true"
+COPY src/requirements.txt ./
 RUN pip install -r requirements.txt --no-cache-dir
```

Comment:
...here you can pass both files:

`RUN pip install -r requirements.txt -r test-requirements.txt --no-cache-dir`